Dockerfile

```diff
@@ -12,7 +12,7 @@ WORKDIR /opt/warehouse/src/
 RUN set -x \
     && apt-get update \
     && apt-get install --no-install-recommends -y \
-        libjpeg-dev nasm
```

Comment:
This [avoids compiling cwepb-bin from source](https://github.com/imagemin/cwebp-bin/issues/31#issuecomment-588383966) reducing the time it takes to complete `npm ci`.